### PR TITLE
add desktop version

### PR DESCRIPTION
**Pull request by MateeSaafi**
short summary: 
- create a new nav bar for desktop version 
- position the headline section for desktop

### CODE REVIEW

**Review by Cosmosha — CHANGES_REQUESTED** [review]
Hi @matee-safi, 
Great Work Done, however, you need to fix something small on your code. 

- Please check and fix the stylelinter Error.

File: styles.css [hunk 1]
```diff
@@ -485,3 +485,340 @@ textarea::placeholder {
...
+    grid-column: 1;
+    grid-row: 1;
+    margin-top: 55px;
+    min-width: 100%;
+  }
+
+  .desktop-works {
+    display: grid;
+    width: 100%;
+    grid-template: 100% / 50% 50%;
```
Comment (huseyinbiyik) [inline styles.css]: Great job using grid 👌 But let's stick to the design and get the ratio of 60%/40% for this featured card like on the Figma template below.
Screenshot 2023-01-20 at 8 48 32 AM

File: index.html [hunk 2]
```diff
@@ -137,13 +177,66 @@ <h2>Professional Art<br>Printing Data</h2>
             <button class="seeProject-btn color-primary button">See Project</button>
         </div>
     </div>
+    <div class="work-grids-desktop color-primary">
+        <div class="work-grid-1 work-grid-desktop">
+            <button class="seeProject-btn color-primary button">See project</button>
+        </div>
```
Comment (huseyinbiyik) [inline index.html]: For the cards in the works section, there are two states of the cards. And we need the "See Project" button for each card for sure. Kindly check the image below to understand the default and hover state of the cards and fix your cards accordingly. 👍
Screenshot 2023-01-20 at 8 53 55 AM

File: index.html [hunk 3]
```diff
@@ -167,8 +260,10 @@ <h2 class="color-secondary">About me</h2>
     </div>
     </section>
     <section class="contact">
+        <div class="email-header">
+            <h4 id="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+        </div>
         <div class="email">
-            <h4 id="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in</h4>
             <form action="https://formspree.io/f/xpzewrow" method="post">
```
Comment (huseyinbiyik) [inline index.html:278]: Kindly fix the little alignment issue on the left of the Get in Touch button with the rest of the input areas. 👌
Screenshot 2023-01-20 at 9 13 23 AM

File: index.html [hunk 4]
```diff
@@ -58,22 +94,26 @@ <h4>I am a Software Developer</h4>
...
     <section class="works">
         <div class="recent-works">
             <h2>My Recent Works</h2>
             <hr class="border">
+            <div class="desktop-works">
             <img src="assets/Img Placeholder.svg" alt="yoga stretch">
+            <div class="desktop-projects">
             <h3>Multi-Post Stories</h3>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
```
Comment (huseyinbiyik) [inline index.html:107]: Your web page responding very well to different sizes. But let's solve the small text overlaying issue here to make the page more responsive.
Screenshot 2023-01-20 at 8 45 42 AM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @matee-safi,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝

## Required Changes ♻️
- Your commit messages are descriptive and you use imperative sense. But let's capitalize the first letter of each commit message.
Screenshot 2023-01-20 at 9 19 18 AM
"Update work section to be desktop friendly" is better 👌

_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (MateeSaafi) [reply index.html]: @huseyinbiyik hello sir, could you please help me with this part? I've been stuck for hours on this problem.

Comment (huseyinbiyik) [reply index.html]: Hello @matee-safi, unfortunately, I am not available right now. But you can check this [link](https://github.com/Yosefgeda/My-Portfolio/pull/5) to get inspired.

File: styles.css [hunk 5]
```diff
@@ -203,7 +203,7 @@ nav {
   display: grid;
   height: 386px;
   width: 327px;
-  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.2) 90.94%), url(assets/Img\ Placeholder2.svg);
+  background: url(assets/Img\ Placeholder2.svg);
```
Comment (GedeonTS) [inline styles.css:206]: - [ OPTIONAL] I think it would be better to use % instead of px in order to make your website responsive. :recycle:

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @matee-safi ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good design :+1: 
- Good README file :+1: 
- Good job writing a descriptive commit message as requested by the previous reviewer :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**